internal/provider/provider.go

```diff
@@ -49,7 +49,7 @@ func (p *ProwlarrProvider) Schema(_ context.Context, _ provider.SchemaRequest, r
 				Sensitive:           true,
 			},
 			"authorization": schema.StringAttribute{
-				MarkdownDescription: "Token for token-based authentication with Prowlarr. This is an alternative to using an API key. Set this via the `PROWLARR_AUTHORIZATION` environment variable. One of `authorization` or `api_key` must be provided, but not both.",
+				MarkdownDescription: "This token is passed as an authentication header, complementing the `api_key`. Set it using the `PROWLARR_AUTHORIZATION` environment variable. Both `authorization` and `api_key` are required for complete authentication.",
```

Comment:
This should refer to external authorization tools, since it's not strictly Prowlarr related. Something like.
"Authorization header to be passed when using a third party authentication tool. it can be set via `PROWLARR_AUTHORIZATION` environment variable. `api_key` field is still required."